baseDAO.cabal

```diff
@@ -3,6 +3,8 @@ cabal-version: 2.2
 -- This file has been generated from package.yaml by hpack version 0.33.0.
 --
 -- see: https://github.com/sol/hpack
+--
+-- hash: ac81ebe2db072ca09cf1331eb45df29d8279881c629d3d5ec9c171578f19a0b8
```

Comment:
Meh, it's sad. In case of this repo conflicts are more likely here, I think we don't want this hash.

Okay, let's keep these hashes within this PR, I'll create a new issue about porting our approach to `.cabal` files maintenance from `morley`.

Reply:
#65